src/modals/collectible-modal/collectible-modal.tsx

```diff
@@ -172,14 +173,32 @@ export const CollectibleModal = () => {
     }
   }, [description]);
 
+  const propertiesIndex = segmentValues.findIndex(item => item === SegmentControlNamesEnum.Properties);
+  const disabledOffers = segmentValues.findIndex(item => item === SegmentControlNamesEnum.Offers);
+
+  const isPropertiesSelected = propertiesIndex === segmentControlIndex;
+
+  const collectionName = useMemo(() => {
+    if (isNonEmptyArray(galleries)) {
+      return galleries[0].gallery.name;
+    }
+
+    if (isDefined(collection)) {
+      return collection.name;
+    }
+
+    return 'Unkown collection';
```

Comment:
why do we need an unknown collection name? What if NFT does not have a collection?

Reply:
If we have a broken collectible without details data we gonna show fallback